gradle-scripts/plugins.gradle

```diff
@@ -3,7 +3,6 @@ apply plugin: 'maven'
 apply plugin: 'checkstyle'
 apply plugin: 'pmd'
 apply plugin: 'jacoco'
-apply plugin: 'findbugs'
 
```

Comment:
Why must the spotbugs plugin not be assigned here

Reply:
new Gradle plugins do not require to apply.  https://plugins.gradle.org/plugin/com.github.spotbugs